main/task.py

```diff
@@ -1,8 +1,11 @@
 # coding: utf-8
 
+import base64
 import logging
+import urllib
 
 import flask
+from google.appengine.api import urlfetch
```

Comment:
For OCD reasons, it would be nice to follow the import sorted order:
`from google.appengine.api import mail`
`from google.appengine.api import urlfetch`